client/state/plugins/wporg/reducer.js

```diff
@@ -1,38 +0,0 @@
-/**
- * Internal dependencies
- */
-import {
-	WPORG_PLUGIN_DATA_RECEIVE,
-	FETCH_WPORG_PLUGIN_DATA,
-	SERIALIZE,
-	DESERIALIZE
-} from 'state/action-types';
-
-function updatePluginState( state = {}, pluginSlug, attributes ) {
-	return Object.assign( {},
-		state,
-		{ [ pluginSlug ]: Object.assign( {}, state[ pluginSlug ], attributes ) }
-	);
-}
-
-function reducer( state = {}, action ) {
-	const { type, pluginSlug } = action;
-	switch ( type ) {
-		case WPORG_PLUGIN_DATA_RECEIVE:
-			if ( action.data ) {
-				return updatePluginState( state, pluginSlug, Object.assign( { isFetching: false, fetched: true, wporg: true }, action.data ) );
-			}
-			return updatePluginState( state, pluginSlug, Object.assign( { isFetching: false, fetched: false, wporg: false } ) );
-
```

Comment:
yeah.... there's the case of custom plugins. Each time we get a plugin slug, we check it in .org directory to see if it's public plugin and then fetch the data. If we have tried to fetch the data for a plugin slug, and we haven't found anything, `fetched` would be true but `wporg` would be false (we know we have checked in the plugins directory and the plugin is not there)